gitlab/assets/dashboards/overview.json

```diff
@@ -0,0 +1,419 @@
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 15,
+            "x": 1,
+            "y": 12
+        },
+        {
+            "height": 13,
+            "id": 6,
+            "legend": false,
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:gitlab.rack.http_requests_total{*} by {method}.as_count()",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "bars"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": "16",
+            "title_text": "Http requests",
+            "type": "timeseries",
+            "width": 43,
+            "x": 34,
+            "y": 13
+        },
+        {
+            "bgcolor": "gray",
+            "font_size": "18",
+            "height": 10,
+            "html": "Load Information",
+            "id": 7,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 43,
+            "x": 34,
+            "y": 1
+        },
+        {
+            "height": 13,
+            "id": 8,
+            "legend": false,
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:gitlab.pipelines_created_total{*}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "bars"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": "16",
+            "title_text": "Pipelines Created",
+            "type": "timeseries",
+            "width": 43,
+            "x": 79,
+            "y": 29
+        },
+        {
+            "height": 13,
+            "id": 9,
+            "legend": false,
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "sum:gitlab.transaction.event_create_repository_total{*}",
```

Comment:
- ditto template variables
- Shouldn't that be a count suffixed with `as_count()`?